doc/classes/TextServerAdvanced.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TextServerAdvanced" inherits="TextServer" version="4.0">
```

Comment:
Shouldn't this one be in `modules/text_server_adv/doc_classes/`? This needs to be listed manually in `config.py` for that.
Same for `TextServerFallback` in `modules/text_server_fb/doc_classes/`.